moveit_commander/test/python_moveit_commander_ns.py

```diff
@@ -95,7 +95,6 @@ def test_validation(self):
         plan3 = self.plan(current)
         self.assertTrue(self.group.execute(plan3))
 
-
```

Comment:
According to PEP8, this double linebreak was intended/required.

Reply:
Thanks!